src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.cs

```diff
@@ -120,7 +120,26 @@ public Bitmap GetThumbnailImpl(uint cx)
                 var a = await _browser.ExecuteScriptAsync($"document.getElementsByTagName('svg')[0].viewBox;");
                 if (a != null)
                 {
-                    await _browser.ExecuteScriptAsync($"document.getElementsByTagName('svg')[0].style = 'width:100%;height:100%';");
+                    // Retrieve the entire HTML content
+                    var htmlContent = await _browser.ExecuteScriptAsync("document.documentElement.outerHTML;");
```

Comment:
Why do we have to run JavaScript to get these contents?
Aren't we supposed to have that already from other parts in the code? Thinking `SvgContents`.
Makes no sense to asynchronously run JavaScript to get something we should already have.

Reply:
Done.

This issue was not observed in svg preview.
But the same error was seen in Peek. Since a common structure is not used, it would be more accurate to examine it in a separate PR.